edx_rbac/utils.py

```diff
@@ -89,6 +93,9 @@ def request_user_has_implicit_access_via_jwt(decoded_jwt, role_name, context=Non
             return True
         else:
             return feature_roles[role_name] == context
+    elif role_name in system_wide_roles and context == ALL_ACCESS_CONTEXT:
```

Comment:
This is to handle the access for `enterprise_openedx_operator` role. operator roles don't have context attached to them. Please [see](https://github.com/edx/edx-enterprise-data/pull/116/files#r271807400)

Reply:
Also for the `enterprise_openedx_operator` role `if role_name in feature_roles:` will be not executed.

Reply:
the enterprise_openedx_operator role should also map to feature specific roles, but would have a context of `*`, so that we are only checking feature specific roles, which is the purpose of having the mapping.